ouroboros-consensus/docs/Testing.md

```diff
@@ -307,6 +307,42 @@ far behind the wallclock, we might not have sufficient knowledge to translate
 the wallclock to a `SlotNo`, although we _can_ always translate the `SlotNo`
 at the tip of the chain to a `UTCTime`.
 
+
+## Golden tests
+
+Golden tests are specific to each era:
```

Comment:
Should we add link to https://ro-che.info/articles/2017-12-04-golden-tests here? Just for the sake of completion?

Reply:
Good point. I think so. @amesgen ? Any objections? 

The only concern might be this link becomes stale at some point. Maybe we could add some broken links checkers to mitigate this. What do you think?

Reply:
What do you guys think about this one https://github.com/input-output-hk/ouroboros-network/pull/4069/commits/5ba0517b78abec68d3bf57827939e96347c88e5b